### ISSUE TITLE
Suggestion: Add confirmation for date in the past

### ISSUE THREAD

**Issue by lenamtl** [projectsend/projectsend#63]
Expiration Date
Add confirmation notification for date if selected in the past.
This is always a good idea to have a confirmation if user select a date in the past.

**Comment by mschop** [comment]
@lenamtl wouldn't it make more sense to prevent dates in the past? I don't know a single usecase, where an expiration date in the past would make sense.

**Comment by lenamtl** [comment]
I agree with you but I had a case sometimes ago, that admin set some file with expired date then update the date later.
I think this is more flexible to just alert the user, also less code to change as if you want to completely prevent it... you  also have to created a server side validation.

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
